test/SlowTests/Issues/RavenDB-20145.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FastTests;
+using Xunit;
+using Raven.Client.Documents.Indexes;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_20145 : RavenTestBase
+    {
+        public RavenDB_20145(ITestOutputHelper output) : base(output)
+        {
+
+        }
+
+        [Fact]
+        public void CanCreateIndex()
+        {
+            using var store = GetDocumentStore();
+            new CustomIndex().Execute(store);
```

Comment:
Maybe let's index a few documents additionally to make sure we don't get any indexing errors as well